packages/next/next-server/lib/router/utils/route-regex.ts

```diff
@@ -1,4 +1,4 @@
-interface Group {
+export interface Group {
```

Comment:
Is this type being used outside this file?

Reply:
Not directly, it seems TypeScript needs it to be exported if used implicitly through `ReturnType<>`

Reply:
Makes sense, that may be a TS requirement when for generating `.d.ts` files.